acc_test/data/kafka_connect_create_cluster.tf

```diff
@@ -0,0 +1,119 @@
+# kafka connect acceptance test. Using the same base kafka cluster as kafka user testing to minimize cluster creation
+provider "instaclustr" {
+    username = "%s"
+    api_key = "%s"
+    api_hostname = "%s"
+}
+
+resource "instaclustr_cluster" "kafka_cluster" {
```

Comment:
Is this used at all? Looks like maybe this tests reuse some other Kafka Cluster?

Reply:
so this one is used for the Kafka Connect clusters. And you are right, this Kafka cluster is reused from the other tests (user, ACL, and topic)

Reply:
so why is it written here again as a unique resource? SHouldn't it exist somewhere else already

Reply:
refactored the `kafka_cluster` out and now Kafka resource tests refer to the cluster that is read from `kafka_user_create_cluster.tf`